lib/projects/project_member.ex

```diff
@@ -0,0 +1,40 @@
+defmodule GlassFactoryApi.Projects.ProjectMember do
+  @moduledoc """
+  Defines a Member of a Project.
+  """
+
+  alias GlassFactoryApi.Projects.ProjectMember
+
+  @type t() :: %ProjectMember{
+          id: String.t(),
+          user_id: String.t(),
```

Comment:
Kind of weird that ids are strings here and integers in other places 🤔

Reply:
I'm changing where we have integers to strings 😅